trieste/acquisition/multiobjective/analytic.py

```diff
@@ -0,0 +1,161 @@
+from itertools import product
+from math import inf
+from typing import Callable, Union
+
+import tensorflow as tf
+from tensorflow_probability import distributions as tfd
+
+from ...data import Dataset
+from ...models import ProbabilisticModel
+from ...type import TensorType
+from ...utils.pareto import Pareto
+from ..function import AcquisitionFunction
+from .function import HypervolumeAcquisitionBuilder, get_reference_point
+
+
+class Expected_Hypervolume_Improvement(HypervolumeAcquisitionBuilder):
```

Comment:
```suggestion
class ExpectedHypervolumeImprovement(HypervolumeAcquisitionBuilder):
```
class names use camel case format in Python